buildSrc/src/main/groovy/io/micronaut/guides/GuideProjectGenerator.groovy

```diff
@@ -60,6 +60,10 @@ class GuideProjectGenerator implements AutoCloseable {
 
         mergeMetadataList(metadatas)
 
+        if (metadatas.publish && !metadatas.category) {
```

Comment:
`metadatas.publish` and `metadatas.category` are lists (shorthand for `*.`) so this will always be false since `metadatas.category` won't be empty

Reply:
No idea what I was thinking.